---
title: "DNA barcodes and reliable molecular identifications in a diverse group of invasive pests: lessons from Bactrocera fruit flies on variation across the COI gene, introgression, and standardization"
authors:
- admin
- Michael San Jose
- Luc Leblanc
- Norman Barr
- Scott Geib
- Arthur Chung
- Julian Dupuis
- Arni Ekayanti
- Elaida Fiegalan
- K.S. Hemachandra
- M. Aftab Hossain
- Chia-Lung Huang
- Yu-Feng Hsu
- Kimberly Morris
- Andi Maryani Mustapeng
- Jerome Niogret
- Thai Hong Pham
- U.G.A.I. Sirisena
- Daniel Rubinoff
author_notes: ""
date: "2020-11-23T00:00:00Z"
doi: "https://doi.org/10.1101/2020.11.23.394510"

# Schedule page publish date (NOT publication's date).
publishDate: "2017-01-01T00:00:00Z"

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["2"]

# Publication name and optional abbreviated publication name.
publication: "ZooKeys"
publication_short: ""

abstract: "The utility of a universal DNA ‘barcode’ fragment of 658 base pairs of the Cytochrome C Oxidase I (COI) gene for the recognition of all animal species has been a widely debated topic on theoretical and practical levels. Regardless of its challenges, large amounts of COI sequence data have been produced in the last two decades. To optimally use the data towards reliable species identification will require further steps to validate the method and reference libraries. The fruit fly tribe Dacini holds about a thousand species, of which eighty are pests of economic concern, including some of the world’s foremost fruit and vegetable pests, and there are many morphologically cryptic species complexes in the tribe. Where previous studies showed limited success in using COI to identify Dacini, our results with a highly curated morphological dataset indicate high congruence between morphology and COI: 90% of the species in our 5,576 sequences, 262-species global dataset can be identified with COI alone based on a monophyly criterion. However, in some key pest species belonging to complexes that were previously thought diagnosable with COI, we found that expanded sampling and independent validation of identifications using genomic data revealed introgression of mitochondrial DNA. We find that the informative SNPs are uniformly distributed across the COI gene, and we provide recommendations for standardization. We conclude that reliable molecular identifications with COI require extensive species coverage, population sampling, and genomics-supported reference identifications before they can be validated as a “diagnostic” marker for specific groups."

# Summary. An optional shortened abstract.
summary: "Where previous studies showed limited success in using COI to identify Dacini, our results with a highly curated morphological dataset indicate high congruence between morphology and COI: 90% of the species in our 5,576 sequences, 262-species global dataset can be identified with COI alone based on a monophyly criterion."

tags:
- Source Themes
featured: true

# links:
# - name: ""
#   url: ""
url_pdf: 'https://www.biorxiv.org/content/10.1101/2020.11.23.394510v1.full.pdf'
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
image:
  caption: 'Image credit: [C Doorenweerd](https://unsplash.com/photos/jdD8gXaTZsc)'
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: example
---
